python/desc/twinkles/sqlite_tools.py

```diff
@@ -29,3 +31,30 @@ def create(self, columns, table, condition=None):
             query += ' ' + condition
         data = [row for row in self.conn.execute(query)]
         return pd.DataFrame(data=data, columns=columns)
+
```

Comment:
At some point, the user does need to understand the content of the data they are using, so since the fact that the duplicates exist is documented, I think that's sufficient.